<launch>
 <!-- Turn on hybrid_astar node -->
 <node name="hori_compar" pkg="hybrid_astar" type="hori_compar.py" output="screen">
 </node>

 <node name="tf_broadcaster" pkg="hybrid_astar" type="tf_broadcaster" />
 <node name="occupancy_server" pkg="grid_map_demos" type="occupancy_server.py" args="$(find hybrid_astar)/maps/occ-map-2019-07-18-19:54:41.yaml" />
 <node name="rviz" pkg="rviz" type="rviz" args="-d $(find hybrid_astar)/launch/hori_compar.rviz" />
 <node name="spline" pkg="hybrid_astar" type="spline.py"  output="screen" />
 <node name="euler_from_quaternion" pkg="hybrid_astar" type="euler_from_quaternion.py"  output="screen" />
 <node name="quaternion_from_euler" pkg="hybrid_astar" type="quaternion_from_euler.py"  output="screen" />
 <node name="feature_extraction" pkg="hybrid_astar" type="feature_extraction.py"  output="screen" />
 <node name="motion_prediction" pkg="hybrid_astar" type="motion_prediction.py"  output="screen" />
</launch>
